Formula/apache-arrow-glib.rb

```diff
@@ -6,6 +6,7 @@ class ApacheArrowGlib < Formula
   sha256 "be0342cc847bb340d86aeaef43596a0b6c1dbf1ede9c789a503d939e01c71fbe"
   license "Apache-2.0"
   head "https://github.com/apache/arrow.git"
+  revision 1
```

Comment:
This isn't needed. Not everyone needs to get the chance right now because it doesn't change anything for the users.

Reply:
Ok, removed the revision directive

Reply:
Sorry for the noise.